mmil.raw.html

```diff
@@ -12302,6 +12306,419 @@
 
 </TABLE>
 
+<h2 style="border-top: 1px solid black; color: #006633; font-weight: bold; margin-top: 20px; padding-top: 10px; font-size: 100%">
+  Metamath 100 status
+</h2>
+
+<p>The <a href="../mm_100.html" >Metamath 100</a> theorems, especially
+those already proved in set.mm, provide one indication of how complete
+iset.mm is.  For each theorem it should be possible to prove it from
+our axioms or show that it cannot be proved (for example, showing that
+it implies excluded middle).  In some cases the natural statement of
+the theorem may need to be chosen (for example, "irrational" as
+used in ~ sqrt2irrap is defined in terms of apartness).</p>
+
+<p>The Metamath 100 page lists those theorems which have been
+proved, but here are some notes on those which have been
+proved in set.mm but not iset.mm:</p>
+
+<table BORDER CELLSPACING=0 BGCOLOR="#EEFFFA">
+  <tr>
+    <th>Theorem</th>
+    <th>notes</th>
+  </tr>
+
+  <tr>
+    <td>2.  The Fundamental Theorem of Algebra</td>
+    <td>The [Geuvers] reference describes a proof
+    without excluded middle, and presumably
+    could be used as our guide.</td>
+  </tr>
+
+  <tr>
+    <td>4.  Pythagorean Theorem</td>
+    <td>There are a variety of theorems
+    which might plausibly deserve the label of
+    "Pythagorean Theorem" but assuming we stick
+    with pythi from set.mm, there are a number of
+    prerequisites.</td>
```

Comment:
4 and 78 are in an obsolete section, so presumably the set.mm proofs will have to be revised first.

Reply:
Exactly how closely an iset.mm proof is likely to follow the set.mm one will vary, but in many cases this kind of improvement to set.mm will make it a lot easier.

Reply:
> 4 and 78 are in an obsolete section, so presumably the set.mm proofs will have to be revised first.

@icecream17 thank you for this hint. I just created two issues #4216 and #4217 for such a revision.